examples/playground/app.tsx

```diff
@@ -72,7 +72,7 @@ autocomplete<ProductHit>({
       {
         sourceId: 'products',
         getItems() {
-          return getAlgoliaHits<ProductHit>({
+          return getAlgoliaResults<ProductRecord>({
```

Comment:
Fixed in https://github.com/algolia/autocomplete/pull/542

Reply:
I think the problem is that [`client.search` returns `Hit<TRecord>`](https://github.com/algolia/algoliasearch-client-javascript/blob/cd39284a204a8bf2f6a99220bf96f0298b12c9c5/packages/client-search/src/types/SearchResponse.ts#L9), so if we provide `ProductHit` (and not `ProductRecord`) to `getAlgoliaResults`, it would return `Hit<ProductHit>` (i.e., `Hit<Hit<ProductRecord>>`). Perhaps that's not an issue since `Hit` is idempotent, but not too sure how TypeScript handles those.

So providing `ProductHit` indeed seems more correct, although then we're manipulating `Hit<ProductHit>` everywhere else.